src/components/header/header.js

```diff
@@ -1,13 +1,29 @@
-import { HeaderLeft, HeaderLink, HeaderLogo, HeaderRight, HeaderSection, HeaderTitle, HeaderWork } from "./styledHeader"
+import { Hamburger, HeaderLeft, HeaderLink, HeaderLogo, HeaderNav, HeaderRight, HeaderSection, HeaderTitle, HeaderWork } from "./styledHeader"
 import logo from '../../images/logo.svg';
 import logoT from '../../images/logoTipped.svg';
+import hamburger from '../../images/hamburger.svg';
+import menuExit from '../../images/menuExit.svg';
 import { useState } from 'react';
-import { headerLinks } from '../../constants/header';
+import { headerLinks } from '../../constants/headerConst';
+import Menu from "../menu/Menu";
+import uniqueId from 'lodash.uniqueid';
 
 const Header = () => {
 
     const [image, setImage] = useState(logo);
 
+    const [menuOpen, setMenuOpen] = useState(false);
+
+    const toggleMenu = () => {
+        setMenuOpen(!menuOpen);
+    }
+
+    window.addEventListener('resize', function () {
```

Comment:
That's cool, i didn't know there was a resize event. Everything seems to work but it might be best practice to put this in a useEffect so the event listener adds when the header mounts, then remove the event listener when the header dismounts. I'll send you a picture in slack.